client/src/components/Booker.js

```diff
@@ -23,8 +20,8 @@ class Booker extends React.Component {
     if (window.location.pathname !== '/') {
       id = window.location.pathname;
     }
-
-    axios.get(`http://booking-dev2.us-west-1.elasticbeanstalk.com/booking/dates${id}`)
+    //http://booking-dev2.us-west-1.elasticbeanstalk.com
+    axios.get(`http://localhost:3004/booking/dates${id}`)
```

Comment:
this is a extra credit nonsense comment: but you can have this as a variable and switch the variable depending on if you're in node_env production or development. that is, when you're in development mode, it'll use local host and in production mode it'll use the elasticbeanstalk address. I think scully is doing it this way and I have a note to refactor to be able to use production/development "modes"